pkg/cli/server/config_reloader_test.go

```diff
@@ -508,9 +510,12 @@ func TestConfigReloader(t *testing.T) {
 		So(string(data), ShouldContainSubstring, "\"UpdateInterval\":18000000000000")
 		So(string(data), ShouldContainSubstring, "\"Scrub\":null")
 		So(string(data), ShouldContainSubstring, "\"DBRepository\":\"another/unreachable/trivy/url2\"")
-		// matching log message when it errors out, test that indeed the download will try the second url
+
+		// Just verify the new URL appears in the logs to confirm config reload worked and ignore
+		// the order of json message formatting that can change independent of this functional
+		// test.
 		found, err = test.ReadLogFileAndSearchString(logFile.Name(),
-			"\"dbRepository\":\"index.docker.io/another/unreachable/trivy/url2:2\",\"goroutine", 1*time.Minute)
+			"index.docker.io/another/unreachable/trivy/url2", 1*time.Minute)
```

Comment:
```suggestion
			"\"dbRepository\":\"index.docker.io/another/unreachable/trivy/url2:2\", 1*time.Minute)
```
otherwise it also matches the other log message also checked above in `data`

Reply:
Leave it as is. The "failed to..." is already checked above. I don't know why we would have a second wait in a loop for the same thing... Your initial change should have worked too...
Maybe the test was patched a few times without a serious review/refactor.

This is good from my POV.